admin/client.go

```diff
@@ -691,9 +691,15 @@ func (e *GenericOpenAPIError) SetError(errorString string) {
 	e.error = errorString
 }
 
-// format error message using title and detail when model implements Error
-func formatErrorMessage(status, path, method string, v ApiError) string {
-	return fmt.Sprintf("%v %v: HTTP %v (Error code: %q) Detail: %v Reason: %v. Params: %v",
+// FormatErrorMessageWithDetails: format error message using error struct fields: Should be only used for testing
+func FormatErrorMessageWithDetails(status, path, method string, v ApiError) string {
```

Comment:
can't it be used in prod code, e.g. to show Terraform errors to the user?

Reply:
https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_2_error_handling.md 

With that context I do not think we need anyone to use that method directly. Exposing that method will give us ability to do better unit tests in cli/TF as we can autoformat error message (right now we have those values hardcoded which is maintenance burden every time test changes).

Reply:
ok